e2e/cloud-plugins-suite/azure-monitor.spec.ts

```diff
@@ -171,14 +171,10 @@ e2e.scenario({
         zone: 'Coordinated Universal Time',
       },
     });
-    e2e()
```

Comment:
Ugh, not totally fixed yet...

Reply:
I think it *could* be failing because you'll have to wait for one of the other search related requests. This stuff is finicky :/

Reply:
Sorted it I think :crossed_fingers:. Removed most e2e tests related to regions. Also fixed one test that I'm not sure was passing to begin with? :man_shrugging: